charts/consent/templates/deployment.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}-deployment
+  labels:
+{{ include "consent.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicas }}
+  revisionHistoryLimit: 0
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+  template:
+    metadata:
+      name: {{ .Chart.Name }}
+      labels:
+{{ include "consent.labels" . | indent 8 }}
+        app: {{ .Chart.Name }}
```

Comment:
can we just add `app: {{ .Chart.Name }}` to the consent labels template. We should make this change for ontology too. I think it was  like this for ontology because we added the template labels later

Could also just `app.kubernetes.io/component: {{ .Chart.Name }}` in `matchLabels` on the selector